.github/actions/smoke-test/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+TEMPLATE_ID="$1"
+
+set -e
```

Comment:
Please use:

```
set -o errexit
set -o nounset
```

and I think it should be first, prior to setting the TEMPLATE_ID.